RpcParser/README.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+The markdown generator script parses the `MOBILE_API.xml` specification and creates a human-readable markdown file for the API.
+
+## Dependencies
+
+The markdown generator requires the use the interface parser included in this project, which has its own set of dependencies. These dependency libraries are described in the interface parser's `requirements.txt` and should be pre-installed by the following command:
+
+```bash
+pip3 install -r ../InterfaceParser/requirements.txt
```

Comment:
One minor suggestion. pip may not be installed by default on the system so we might want to include a pip installation step (like we did for the HMIs https://github.com/smartdevicelink/generic_hmi#dependencies)

```suggestion
sudo apt install python3 python3-pip -y
pip3 install -r ../InterfaceParser/requirements.txt
```